src/tests.rs

```diff
@@ -45,6 +46,23 @@ fn test_rpm_builder() -> Result<(), Box<dyn std::error::Error>> {
 
     pkg.verify_digests()?;
 
+    // check various metadata on the files
+    pkg.metadata.get_file_entries()?.iter().for_each(|f| {
+        if f.path.as_os_str() == "/etc/awesome/second.toml" {
+            assert_eq!(
+                f.clone().caps.unwrap(),
+                "cap_sys_ptrace,cap_sys_admin=ep".to_string()
+            );
+            assert_eq!(f.ownership.user, "hugo".to_string());
+        } else if f.path.as_os_str() == "/etc/awesome/config.toml" {
+            // The "=" capability is equivalent to an empty capability set
```

Comment:
Not sure if this comment is still relevant, but also it's not harmful

Reply:
Removed